extensions/PhabBugz/Extension.pm

```diff
@@ -54,6 +59,48 @@ sub webservice {
   $args->{dispatch}->{PhabBugz} = "Bugzilla::Extension::PhabBugz::WebService";
 }
 
+sub object_end_of_update {
+  my ($self,   $args)    = @_;
+  my ($object, $changes) = @$args{qw(object changes)};
+  my $params = Bugzilla->params;
+
+  next
+    if !$params->{phabricator_enabled}
+    || !$object->isa('Bugzilla::User')
+    || !$changes
+    || !$changes->{disabledtext};
+
+  my $user = $object;
+
+  my $orig_error_mode = Bugzilla->error_mode;
+  Bugzilla->error_mode(ERROR_MODE_DIE);
+
+  try {
+    my $phab_user
+      = Bugzilla::Extension::PhabBugz::User->new_from_query({ids => [$user->id]});
+    return if !$phab_user;
+    $phab_user->set_user_enabled_status(($user->is_enabled ? 1 : 0));
```

Comment:
Just curious: why are you passing in `0`/`1` instead of `$user->is_enabled` directly?

Reply:
Good point. Just figured it would be more clear the to the viewer but it is technically redundant. I will change that.